website/docs/docs/deploy/job-scheduler.md

```diff
@@ -82,6 +83,23 @@ The scheduler prevents queue clog by canceling runs that aren't needed, ensuring
 
 To prevent over-scheduling, users will need to take action by either refactoring the job so it runs faster or modifying its [schedule](/docs/deploy/deploy-jobs#schedule-days).
 
+## Deactivation of abandoned jobs
+
+The scheduler will consider a job abandoned if it meets these criteria: 
+- It's had 10 or more consecutive run errors
+- It's been 10 or more days since its last successful run
+- No code changes to the branch since the first failure
+
+Abandoned jobs, typically, are not used by anyone and with no one monitoring them becomes a performance drain and load balancing issue for the scheduler. To mitigate this situation, the scheduler will deactivate a job if it meets the defintion for abandoned and the job will no longer be enqueued to run. 
```

Comment:
Suggest a small rephrase/reorder of the segment:
```suggestion
dbt Cloud automatically deactivates abandoned jobs to reduce unnecessary resource consumption and reduce contention for run slots in your account. A job is considered abandoned if it meets all of the following criteria:
- It's had 10 or more consecutive run errors
- It's been 10 or more days since its last successful run
- No code changes have been committed to the main branch since the first failure
```
main branch? default branch? idk what our name for it is offhand.